library/build.gradle

```diff
@@ -73,6 +73,7 @@ dependencies {
     implementation "androidx.palette:palette-ktx:$paletteKtxVersion"
 
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycleVersion"
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:$lifecycleVersion"
```

Comment:
No need to add this one. I already mentioned a solution in our conversation under `runBlocking`

Reply:
updated.